firmware/application/apps/ui_mictx.cpp

```diff
@@ -475,6 +480,17 @@ MicTXView::MicTXView(
         rogerbeep_enabled = v;
     };
 
+    check_mic_to_HP.on_select = [this](Checkbox&, bool v) {
+        mic_to_HP_enabled = v;
+        (mic_to_HP_enabled ? audio::input::loopback_mic_to_hp_enable() : audio::input::loopback_mic_to_hp_disable());
```

Comment:
I guess this is valid, but I've never seen the conditional/ternary operator used like this (where the result of the expression is discarded).  Personally, if the result of the expression is to be discarded, I would write it as an "if" statement instead, i.e.:
```
if (mic_to_HP_enabled)
    audio::input::loopback_mic_to_hp_enable();
else
    audio::input::loopback_mic_to_hp_disable();
```
Or, even more optimally:

`audio::input::loopback_mic_control(mic_to_HP_enabled);`

Just a thought.  Same suggestion applies to the 4 other lines like this as well.

Reply:
Well , the compiler did not give me  any warning about it , and googling it , it seems that this syntax is allowed in C+ (and I believed that also in C++).   Anyway you are the sw specialist ,not me .  if the general consensus is that I should better change it , I have no objection at all . Just let me know and I will proceed changing it .